### PR TITLE
ISSD-3256 Adjust front end to add archived date

### PR DESCRIPTION
**Pull request by rafaelfs98**

### CODE REVIEW

**Comment by rafaelfs98** [comment]
## :fire: GitRay Report 
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
 ## Jira Tasks 

 <html></html> 

---

Hey, don't forget to update all related tickets with correct status, such as **In Review** :wink: 
### Git Stats 

4 files changed, 27 insertions(+), 1 deletion(-)

**Comment by rafaelfs98** [comment]
ci:test:sf

File: workspaces/liferay-testray-workspace/client-extensions/liferay-testray-site-initializer/site-initializer/object-definitions/testray-build.json [hunk 1]
```diff
@@ -15,6 +15,17 @@
...
+		{
+			"DBType": "Date",
+			"indexed": true,
+			"indexedAsKeyword": false,
+			"label": {
+				"en_US": "Date Arquived"
+			},
+			"name": "startDate",
+			"required": false,
+			"type": "Date"
```
Comment (JoMaAlves) [inline workspaces/liferay-testray-workspace/client-extensions/liferay-testray-site-initializer/site-initializer/object-definitions/testray-build.json:27]: Is this right? I saw that you added 2 fields with the label "Date Arquived", but one has "startDate" as name and the other "dateArchived". Could you review it?

File: workspaces/liferay-testray-workspace/client-extensions/liferay-testray-site-initializer/site-initializer/object-definitions/testray-build.json [hunk 2]
```diff
@@ -59,6 +70,17 @@
...
+			"DBType": "Date",
+			"indexed": true,
+			"indexedAsKeyword": false,
+			"label": {
+				"en_US": "Date Archived"
+			},
+			"name": "dateArchived",
+			"required": false,
+			"type": "DateTime"
+		},
```
Comment (JoMaAlves) [inline workspaces/liferay-testray-workspace/client-extensions/liferay-testray-site-initializer/site-initializer/object-definitions/testray-build.json:83]: The other "Date Archived", check the first comment.

**Review by JoMaAlves — CHANGES_REQUESTED** [review]
🛠️ Needs Review 🛠️
I think there was a mistake there, could you give it a look @rafaelfs98 ?

**Review by JoMaAlves — APPROVED** [review]
LGTM ​🦖​🌋​ 
Thank you @rafaelfs98

**Comment by joseabelenda** [comment]
resent:https://github.com/brianchandotcom/liferay-portal/pull/148585

**Pull request closed without merging**